<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::Dialog::DlgSettingsGeneral</class>
 <widget class="QWidget" name="Gui::Dialog::DlgSettingsGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h2 align=&quot;center&quot; style=&quot; margin-top:16px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600;&quot;&gt;Welcome to Ondsel ES&lt;/span&gt;&lt;/h2&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QComboBox" name="Languages">
       <property name="toolTip">
        <string>Language of the application's user interface</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="unitSystemLabel">
       <property name="text">
        <string>Default Unit system:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="iconSizeLabel">
       <property name="text">
        <string>Size of toolbar icons:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="TextLabel1_4">
       <property name="text">
        <string>Language:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="Gui::PrefComboBox" name="comboBox_UnitSystem">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Unit system for all parts of the application. Can be overridden by specifying a document unit system.</string>
       </property>
       <property name="prefEntry" stdset="0">
        <cstring>UserSchema</cstring>
       </property>
       <property name="prefPath" stdset="0">
        <cstring>Units</cstring>
       </property>
       <item>
        <property name="text">
         <string>Standard (mm, kg, s, °)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MKS (m, kg, s, °)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>US customary (in, lb)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Imperial decimal (in, lb)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Building Euro (cm, m², m³)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Building US (ft-in, sqft, cft)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Metric small parts &amp; CNC(mm, mm/min)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Imperial for Civil Eng (ft, ft/s)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FEM (mm, N, s)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Meter decimal (m, m², m³)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="navigationLabel">
       <property name="text">
        <string>3D Navigation Style</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="Gui::PrefComboBox" name="toolbarIconSize">
       <property name="toolTip">
        <string>Choose your preference for toolbar icon size. You can adjust
this according to your screen size or personal taste</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="prefEntry" stdset="0">
        <cstring>ToolbarIconSize</cstring>
       </property>
       <property name="prefPath" stdset="0">
        <cstring>General</cstring>
       </property>
       <item>
        <property name="text">
         <string>Small (16px)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Medium (24px)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Large (32px)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Extra Large (48px)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="Gui::PrefComboBox" name="comboNavigationStyle">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Navigation settings set</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="prefEntry" stdset="0">
        <cstring>NavigationStyle</cstring>
       </property>
       <property name="prefPath" stdset="0">
        <cstring>View</cstring>
       </property>
       <item>
        <property name="text">
         <string>Blender</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CAD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gesture</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MayaGesture</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OpenCascade</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OpenInventor</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OpenSCAD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Revit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TinkerCAD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Touchpad</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="3" rowspan="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" rowspan="4">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Gui::PrefComboBox</class>
   <extends>QComboBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
